src/azure-cli/azure/cli/command_modules/appconfig/_featuremodels.py

```diff
@@ -167,6 +171,48 @@ def custom_serialize_conditions(conditions_dict):
     return featurefilterdict
 
 
+def map_featureflag_to_keyvalue(featureflag):
+    '''
+        Helper Function to convert FeatureFlag object to KeyValue object
+
+        Args:
+            featureflag - FeatureFlag object to be converted
+
+        Return:
+            KeyValue object
+    '''
+    try:
+        enabled = False
+        if featureflag.state in ("on", "conditional"):
+            enabled = True
+
+        feature_flag_value = FeatureFlagValue(id_=featureflag.key,
+                                              description=featureflag.description,
+                                              enabled=enabled,
+                                              conditions=featureflag.conditions)
+
+        set_kv = KeyValue(key=FEATURE_FLAG_PREFIX + featureflag.key,
+                          label=featureflag.label,
+                          value=json.dumps(feature_flag_value,
+                                           default=lambda o: o.__dict__,
+                                           ensure_ascii=False),
+                          content_type=FEATURE_FLAG_CONTENT_TYPE,
+                          tags={})
+
+        set_kv.locked = featureflag.locked
+        set_kv.last_modified = featureflag.last_modified
```

Comment:
These two are server generated. No need to map to KeyValue. As in KeyValue, etag/last_modified/locked are not in ctor parameters. Although in Python basically there is no private property, it won't be included in request body and will aslo be ignored by backend.